### PR TITLE
docs: add CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by joshuajames-smith**
## Description

Large documentation addition of CODE_OF_CONDUCT.md which improves the gliff.ai GitHub Community Health Files. 

*(note: file name uses `_` instead of `-` to meet GitHub's requested format which also allows for the documentation to be recognised by the system)

### CODE REVIEW

**Review by ChasNelson1990 — COMMENTED** [review]
Should this more closely match our own internal code of conduct? Which is a lot more details - could be good, could be bad?

**Comment by joshuajames-smith** [comment]
> Should this more closely match our own internal code of conduct? Which is a lot more details - could be good, could be bad?

@ChasNelson1990 think having a long complex document might off-put contributors, maybe just need to identify key points from ours and keeping it simplified - assuming the gliff.ai team are really abiding by the internal code of conduct, so this maybe more so contributors?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 ⚠️ **All projects and contributors are governed by this document!** ⚠️
 
-The purpose of this guide is to ensure all contributors and projects within the [gliff.ai](https://gliff.ai) space have a unified Code of Conduct to create a positive and welcoming environment for all.
+The purpose of this guide is to outline [gliff.ai](https://gliff.ai)'s expectations of individuals involved in [gliff.ai](https://gliff.ai) activities. While incidents of unaccpetable behaviour are relatively infrequent, this document serves as a guide to form a clear set of standards to maintain a safe and welcoming space that avoids/prevents behaviours that prevent this, while outlining reporting and disciplinary processes.
+
+[gliff.ai](https://gliff.ai) employees a team of passionate and innovative backgrounds, with the outlook that our compnay, our products and our customers benefit from diveristy of backgrounds and perspectives. We prioritise creating a space which is a safe and welcoming environment for all, regardless of their personal identity or background, as anything other than thise can jeopardise the ability to work and with [gliff.ai](https://gliff.ai).
```
Comment (ChasNelson1990) [inline CODE_OF_CONDUCT.md]: We employee a team of backgrounds do we?

Comment (joshuajames-smith) [reply CODE_OF_CONDUCT.md]: Opps my bad! 🥴

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**